src/wasm/cpp/working.cc

```diff
@@ -1,5 +1,5 @@
-#include "arcs.h"
-#include "example-arcs.h"
+#include <arcs.h>
+#include <working-arcs.h>
 #include <vector>
 
 class StorageParticle : public arcs::Particle {
```

Comment:
Suggest adding a header comment noting what this file aims to be, to help a new reader.

Reply:
Done.